WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -613,6 +615,14 @@ class MySiteViewModel @Inject constructor(
                     onPagesItemClick = this::onPagesItemClick,
                     onFooterLinkClick = this::onPagesCardFooterLinkClick
                 ),
+                activityBuilderParams = ActivityCardBuilderParams(
+                    site = site,
```

Comment:
np: Using named arguments is unnecessary imho when the variable name is equal to the param name. I would even go as far as to make it a Kotlin Coding Standard if anyone would listen to me 😅 

ps: not a change request either.

Reply:
> Using named arguments is unnecessary imho

I haven't thought about it that way and I see your point. I'm leaving this as is because it follows the pattern.